lesson6-questions.txt

```diff
@@ -3,36 +3,64 @@ Questions for Lesson 6
 1. When you enable CRUD operations on a table in the database, you need to add four things
 to your Rails application.  When you generate a scaffold, all four are created.What are they?
 
-
+It is a model, a controller, views, and routes. 
 
 
 2. When you specify a route in config/routes.rb, you need to include certain information.  What
 must be included? In addition, you may specify an additional parameter to create a variable
 that describes the route.  How do you do that? What variable is created?
 
+You need to include:
 
+1. The URL path that the route will match.
```

Comment:
You need another thing paired with the URL. It's similar to the action (method) you mention later.